pages/common/steam.md

```diff
@@ -0,0 +1,12 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://store.steampowered.com/>.
+
+- Launch Steam client in background:
+
+`steam &`
```

Comment:
```suggestion
```
Same as for the other page. It isn't really command specific. 

However I found <https://developer.valvesoftware.com/wiki/Command_Line_Options#Command-line_parameters_3>, which has a good amount of command line options for steam. It says it's for Windows, but I tested some and they all work for Linux too.

Reply:
If you like to, go ahead. I'll convert the PR to a draft in between.

Reply:
sounds good. hopefully i should have it done later today, but in the next couple days at the latest.